### ISSUE TITLE
Release time?

### ISSUE THREAD

**Issue by velo** [mpollmeier/orientdb-gremlin#61]
@mpollmeier ain't time for a new release?

**Comment by mpollmeier** [comment]
I'm on it. 3.1.1-incubating.0

**Comment by mpollmeier** [comment]
I'm getting heaps of errors during the javadoc generation - any idea what this is?

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.9.1:jar (attach-javadocs) on project orientdb-gremlin: MavenReportException: Error while creating archive:
[ERROR] Exit code: 1 - /home/mp/Projects/tinkerpop/orientdb-gremlin/driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientEdge.java:135: warning: no @return
[ERROR] public OIdentifiable getInVertex() {
[ERROR] ^
[ERROR] /home/mp/Projects/tinkerpop/orientdb-gremlin/driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientGraph.java:233: warning: no @param for label
[ERROR] public String labelToClassName(String label, String prefix) {
[ERROR] ^
[ERROR] /home/mp/Projects/tinkerpop/orientdb-gremlin/driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientGraph.java:233: warning: no @param for prefix
[ERROR] public String labelToClassName(String label, String prefix) {
[ERROR] ^
[ERROR] /home/mp/Projects/tinkerpop/orientdb-gremlin/driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientGraph.java:233: warning: no @return
[ERROR] public String labelToClassName(String label, String prefix) {
[ERROR] ^
[ERROR] /home/mp/Projects/tinkerpop/orientdb-gremlin/driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientGraph.java:243: warning: no @param for className
[ERROR] public String classNameToLabel(String className) {
[ERROR] ^
[ERROR] /home/mp/Projects/tinkerpop/orientdb-gremlin/driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientGraph.java:243: warning: no @return
[ERROR] public String classNameToLabel(String className) {
[ERROR] ^
[ERROR] /home/mp/Projects/tinkerpop/orientdb-gremlin/driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientGraphFactory.java:50: warning: no @return
[ERROR] public OrientGraph getNoTx(boolean create, boolean open) {
[ERROR] ^
[ERROR] /home/mp/Projects/tinkerpop/orientdb-gremlin/driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientGraphFactory.java:123: warning: no @return
[ERROR] protected ODatabaseDocumentTx getDatabase(boolean create, boolean open) {
[ERROR] ^
[ERROR] /home/mp/Projects/tinkerpop/orientdb-gremlin/driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientGraphFactory.java:135: error: unknown tag: label
[ERROR] * Enable or disable the prefixing of class names with V_<label> for
[ERROR] ^
[ERROR] /home/mp/Projects/tinkerpop/orientdb-gremlin/driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientGraphFactory.java:136: error: unknown tag: label
[ERROR] * vertices or E_<label> for edges.
[ERROR] ^
[ERROR] /home/mp/Projects/tinkerpop/orientdb-gremlin/driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientGraphFactory.java:142: warning: no @return
[ERROR] public OrientGraphFactory setLabelAsClassName(boolean is) {
[ERROR] ^
[ERROR] /home/mp/Projects/tinkerpop/orientdb-gremlin/driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientGraphFactory.java:151: warning: no @param for max
[ERROR] public OrientGraphFactory setupPool(final int max) {
[ERROR] ^
[ERROR] /home/mp/Projects/tinkerpop/orientdb-gremlin/driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientGraphFactory.java:151: warning: no @return
[ERROR] public OrientGraphFactory setupPool(final int max) {
[ERROR] ^
[ERROR]
[ERROR] Command line was: /usr/lib/jvm/java-8-openjdk/jre/../bin/javadoc @options @packages
[ERROR]
[ERROR] Refer to the generated Javadoc files in '/home/mp/Projects/tinkerpop/orientdb-gremlin/driver/target/apidocs' dir.
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

**Comment by velo** [comment]
It's complaing about  `<label>`
https://github.com/mpollmeier/orientdb-gremlin/blob/master/driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientGraphFactory.java#L135

```
[ERROR] /home/mp/Projects/tinkerpop/orientdb-gremlin/driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientGraphFactory.java:135: error: unknown tag: label
[ERROR] * Enable or disable the prefixing of class names with V_<label> for
[ERROR] ^
[ERROR] /home/mp/Projects/tinkerpop/orientdb-gremlin/driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientGraphFactory.java:136: error: unknown tag: label
[ERROR] * vertices or E_<label> for edges.
[ERROR] ^
```

**Comment by velo** [comment]
Fixed on https://github.com/mpollmeier/orientdb-gremlin/pull/63

**Comment by mpollmeier** [comment]
just released 3.1.1-incubating.0
